runtime/account_test.go

```diff
@@ -1069,84 +1065,6 @@ func TestRuntimePublicKey(t *testing.T) {
 		assert.Contains(t, err.Error(), "value of type `PublicKey` has no member `validate`")
 	})
 
-	t.Run("IsValid", func(t *testing.T) {
-		for _, validity := range []bool{true, false} {
-			script := `
-              pub fun main(): Bool {
-                  let publicKey = PublicKey(
-                      publicKey: "0102".decodeHex(),
-                      signatureAlgorithm: SignatureAlgorithm.ECDSA_P256
-                  )
-
-                  return publicKey.isValid
-              }
-            `
-			invoked := false
-			validateMethodReturnValue := validity
-
-			storage := newTestLedger(nil, nil)
-
-			runtimeInterface := &testRuntimeInterface{
-				storage: storage,
-				validatePublicKey: func(publicKey *PublicKey) (bool, error) {
-					invoked = true
-					return validateMethodReturnValue, nil
-				},
-			}
-
-			value, err := executeScript(script, runtimeInterface)
-			require.NoError(t, err)
-
-			assert.True(t, invoked)
-			assert.Equal(t, cadence.Bool(validateMethodReturnValue), value)
-
-		}
-	})
-
-	t.Run("IsValid - publicKey from host env", func(t *testing.T) {
```

Comment:
Similarly here, can test constructing an invalid `PublicKey` through the fvm-cadence interface.